src/GojsDiagram.test.tsx

```diff
@@ -0,0 +1,150 @@
+import * as React from 'react';
+import * as go from 'gojs';
+import { Diagram } from 'gojs';
+import { mount } from 'enzyme';
+import GojsDiagram from './GojsDiagram';
+
+describe('GojsDiagram component', () => {
```

Comment:
really hard to follow what's happening in this test

Reply:
I will make the description much more specific.
As it is ui component, it's just testing the rendering of the component in different cases described in the it functions

Reply:
I could use the same convention as the enzyme documentation https://github.com/airbnb/enzyme/blob/master/docs/api/shallow.md

Reply:
yes, it's much clearer :) one it per tested behavior